src/com/messagemedia/clientgui/Manager.java

```diff
@@ -164,8 +183,27 @@ public static String sendMessage(SendPanel panel){
     	}catch(Exception e){
     		return "Error: " + e.getMessage();
     	}
-    } 
-    
+    }
+
+	public static String deleteScheduledMessage(SendPanel panel) {
+		try {
+			long uid = 0;
+
+			if (!panel.getUidField().getText().equals("")) {
```

Comment:
I'll reformat the file using just spaces because now there is a mixture of tabs and spaces.